### DISCUSSION TITLE
External Whisper

### DISCUSSION THREAD

**Discussion by ther3zz** [open-webui/open-webui#1606] [Q&A]
Hello, is it currently possible to use an external whisper service instead of loading the model within the open wbeui instance?
I have whisper asr running on a separate machine via docker and I would be cool to use that instance instead of using more resources to accomplish the same thing.

Thanks!

**Comment by ther3zz** [comment]
It would be cool to support the wyoming protocol for use with faster-whisper (the linuxserver image) as well.

**Comment by koolkhel** [comment]
I have external whisper myself, built for Intel GPU etc, so the embedded one wouldn't suffice. Also whisper has its own web api which could be used.

**Comment by mariodiazgomez** [comment]
I must be crazy but once I got asr working by indicating the IP and port to open webui where the model name is entered in whisper (local) and the next day it simply did not work

**Comment by sneak-it** [comment]
Just adding to this old request here.... attempted to get an external `faster-whisper` instance to work here, since I already use it with Home Assistant.

Doesn't appear to be working with these settings:
<img width="1335" height="320" alt="image" src="https://github.com/user-attachments/assets/47d5adae-40b4-4191-89bd-0e1386ee23c3" />

Anyone get this to work?
